docs/spec/v1alpha1/helmcharts.md

```diff
@@ -0,0 +1,142 @@
+# Helm Charts
+
+The `HelmChart` API defines a source for Helm chart artifacts coming
+from [`HelmRepository` sources](helmrepositories.md). The resource
+exposes the latest pulled chart for the defined version as an artifact.
+
+## Specification
+
+Helm chart:
+
+```go
+// HelmChart defines the desired state of a Helm chart.
```

Comment:
```suggestion
// HelmChartSpec defines the desired state of a Helm chart.
```

Also I'm not sure that description is correct. It defines the desired state of a Helm chart _source_, perhaps?